src/components/admin/MainIssueController.jsx

```diff
@@ -173,13 +180,15 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
             if (!has.call(article, 'authors') || !article.authors[0]) {
-              window.alert(
+              this.props.displayAlert(
                 `${article.title} has no authors. Please correct this`,
               );
               return false;
             }
             if (/http(?!s)/.test(article.html)) {
               if (
+                // TODO
```

Comment:
Maybe add what is TODO here in case you don't get around to it.

Reply:
Will do